### ISSUE TITLE
add some bs plugins like cute tees walking on the screen

### ISSUE THREAD

**Issue by ChillerDragon** [discord-irc/discord-irc#39]
This "toy project" has become way to corporate and serious.

Why did I bother to rebuild enterprise scale discord chat applications if not to have a bit of fun as well.
Why did I bother to come up with elaborate plugin systems that allow users to opt out of any bullshit they do not want.

I want some cute twinbop walking on the top of my message bar. Similar to the vscode pets extension
https://marketplace.visualstudio.com/items?itemName=tonybaloney.vscode-pets

Basically like this but it should be moving
![image](https://github.com/discord-irc/discord-irc/assets/20344300/42f43d4b-33df-4067-bd5c-8ff0b38990ac)

**Comment by ChillerDragon** [comment]
Available web libs for skin rendering:

- https://github.com/AlexIsTheGuy/TeeAssembler-2.0
- https://github.com/ddnet/ddnet-web/blob/master/www/skins/tee.js

**Comment by MilkeeyCat** [comment]
the most high priority issue xD

**Comment by MilkeeyCat** [comment]
also you can integrate Chat-GPT4 so twinbop can flirt with you

**Comment by ChillerDragon** [comment]
![image](https://github.com/discord-irc/discord-irc/assets/20344300/ea55e258-db81-49ed-8696-9d22e7717537)

It moves left and right. Not quite as fancy as I envisioned it. But a simple prototype.
